x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/warnings/warnings.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiAccordion, EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { useDatasetQualityWarnings } from '../../../hooks/use_dataset_quality_warnings';
+
+const nonAggregatableWarningTitle = i18n.translate('xpack.datasetQuality.nonAggregatable.title', {
+  defaultMessage: 'Your request may take longer to complete',
+});
+
+const nonAggregatableWarningDescription = (nonAggregatableDatasets: string[]) => (
+  <FormattedMessage
+    id="xpack.datasetQuality.nonAggregatable.description"
+    defaultMessage="{description}"
+    values={{
+      description: (
+        <FormattedMessage
+          id="xpack.datasetQuality.nonAggregatable.warning"
+          defaultMessage="Some of your datasets don't support _ignored aggregation and may cause delays when querying data. {howToFixIt} {showDatasets}"
+          values={{
+            showDatasets: (
+              <FormattedMessage
+                id="xpack.datasetQuality.nonAggregatable.warning.description."
+                defaultMessage="{accordion}"
+                values={{
+                  accordion: (
+                    <EuiAccordion
+                      style={{ marginTop: '12px ' }}
+                      buttonContent={i18n.translate(
+                        'xpack.datasetQuality.nonAggregatable.showAffectedDatasets',
+                        {
+                          defaultMessage: 'Show affected datasets',
+                        }
+                      )}
+                      id={''}
+                    >
+                      <ul>
+                        {nonAggregatableDatasets.map((dataset) => (
+                          <li key={dataset}>{dataset}</li>
+                        ))}
+                      </ul>
+                    </EuiAccordion>
+                  ),
+                }}
+              />
+            ),
+            howToFixIt: (
+              <FormattedMessage
+                id="xpack.datasetQuality.nonAggregatable.howToFixIt"
+                defaultMessage="Manually {rolloverLink} these datasets to prevent future delays."
+                values={{
+                  rolloverLink: (
+                    <EuiLink
+                      external
+                      target="_blank"
+                      data-test-subj="datasetQualityNonAggregatableHowToFixItLink"
+                      href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-rollover-index.html"
```

Comment:
I was initially thinking ideally we could link to a UI page that can trigger a rollover with a button but seems this doesn't exist? As it is temporary for most users, this should be good enough. 

Lets also have some docs around this issue explaining why users see _ignored, that is was not index on previous elasticsearch versions and a rollover makes sure it is index. Then we link to the docs and from there to rollover (or both)? Something we can still iterate on.

Reply:
@mdbirnstiehl can help us in that front. Should we create something like a troubleshooting section? or where should we put that type of information?

Reply:
> Then we link to the docs and from there to rollover (or both)? Something we can still iterate on.

I'd prefer to have the `How to fix it?` visible in the warning, and then some docs if I want to get to the details of why that happened, rather than having to go to the docs to see how to solve it. However don't have a strong opinion in this matter.

Reply:
Having the solution in the warning is good to help users that are aware of rollovers and how they work. Then we can link out to the docs for users that aren't familiar with using the rollover and explain more about why they need to do it. We can probably add a section to the [docs that are in progress currently](https://github.com/elastic/staging-serverless-observability-docs/pull/211). This way we are covering all users and not making advanced users go to the docs necessarily.